# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: third_party/rust/gpu-descriptor/Cargo.toml
# Commit: ef5dc3e04e5f
# Full Hash: ef5dc3e04e5f271eea0636ab3a495e95cc912f1d
# Author: Dzmitry Malyshau <dmalyshau@mozilla.com>
# Date: 2021-09-04 09:40:24
# Regressor Bug: 1726626
# File Overlap Count: 3
# Description:
#   Bug 1726626 - Move gfx/wgpu into a 3rd party dependency r=jgilbert,bholley
#   
#   This update makes wgpu a vendored dependency instead of having it in gfx/wgpu.
#   
#   ## Notes
# ==============================================================================

diff --git a/third_party/rust/gpu-descriptor/Cargo.toml b/third_party/rust/gpu-descriptor/Cargo.toml
--- a/third_party/rust/gpu-descriptor/Cargo.toml
+++ b/third_party/rust/gpu-descriptor/Cargo.toml
@@ -8,21 +8,21 @@
 # If you believe there's an error in this file please file an
 # issue against the rust-lang/cargo repository. If you're
 # editing this file be aware that the upstream Cargo.toml
 # will likely look very different (and much more reasonable)
 
 [package]
 edition = "2018"
 name = "gpu-descriptor"
-version = "0.1.1"
+version = "0.2.1"
 authors = ["Zakarum <zakarumych@ya.ru>"]
 description = "Implementation agnostic descriptor allocator for Vulkan like APIs"
 homepage = "https://github.com/zakarumych/gpu-descriptor"
-documentation = "https://docs.rs/gpu-descriptor-types/0.1.0"
+documentation = "https://docs.rs/gpu-descriptor"
 keywords = ["gpu", "vulkan", "no-std"]
 license = "MIT OR Apache-2.0"
 repository = "https://github.com/zakarumych/gpu-descriptor"
 [dependencies.bitflags]
 version = "1.2"
 default-features = false
 
 [dependencies.gpu-descriptor-types]